code/modules/projectiles/boxes_magazines/internal/grenade.dm

```diff
@@ -15,3 +15,9 @@
 	ammo_type = /obj/item/ammo_casing/caseless/rocket
 	caliber = "84mm"
 	max_ammo = 1
+
+/obj/item/ammo_box/magazine/internal/cannonball
+	name = "cannonball"
+	ammo_type = /obj/item/ammo_casing/caseless/cannonball
+	caliber = "30mm"
```

Comment:
Cannonballs smaller than 1.5 inches were grapeshot and were more like a shotgun, 30mm is 1.1~ inches. should be closer to 100mm (4.1 inches)